### PR TITLE
[3/3 dedupe] Remove restart modal dupe

### PR DESCRIPTION
**Pull request by rcorral**

### CODE REVIEW

File: src/js/components/modals/ServiceRestartModal.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import MarathonStore from '../../stores/MarathonStore';
+import ServiceActionModal from './ServiceActionModal';
+
+class ServiceRestartModal extends ServiceActionModal {
+  constructor() {
+    super();
```
Comment (mlunoe) [inline src/js/components/modals/ServiceRestartModal.js]: `...arguments`

**Comment by mlunoe** [comment]
One #nit, but other than that it looks great!

**Comment by mlunoe** [comment]
Tried to rebase for you, but it was a little more involved. Could you resolve the conflicts on this guy?

**Comment by MatApple** [comment]
@rcorral Flicts

**Comment by mlunoe** [comment]
Tried it out again, works well :) :shipit: Merge, once tests pass

**Pull request merged**